app.py

```diff
@@ -189,7 +195,8 @@ def tootfeed(query_feed):
                                '♻ : ' + str(toot['reblogs_count']) + ', ' + \
                                '✰ : ' + str(toot['favourites_count']) + '</div></blockquote>'
 
-            toot['created_at'] = datetime.datetime.strptime(toot['created_at'], '%Y-%m-%dT%H:%M:%S.%fZ')
+            if isinstance(toot['created_at'], str):
```

Comment:
In the most recent Mastodon.py, `created_at` is already a datetime, so this conversion is only needed with older versions of the client.